ChangeLog.md

```diff
@@ -1,10 +1,4 @@
-
-
-## Backwards-incompatible changes
-
-- Removes APIs deprecated in 0.3.0.0, including `Eff`, `interpret`, `ret`, and the `handle*` family of helper functions.
```

Comment:
I had assumed that the next release would be 0.4—I will restore this text when appropriate! 😄

Reply:
@patrickt: I think this implies you’ll need to revert #143 before merging/shipping this PR too.